python-bindings/b2_terraform/provider_tool.py

```diff
@@ -270,7 +271,13 @@ def resource_update(
 
     def resource_delete(self, *, bucket_id, **kwargs):
         bucket = self.api.get_bucket_by_id(bucket_id)
-        self.api.delete_bucket(bucket)
+        try:
+            self.api.delete_bucket(bucket)
+        except BadRequest as e:
+            if e.code == 'bad_bucket_id':  # bucket was already deleted
```

Comment:
please add a comment that indicates this piece of code forces you to pin to minor version of b2sdk